icarus/egg-icarus--dedicated.json

```diff
@@ -1,10 +1,10 @@
 {
-    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
         "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2024-06-01T00:03:56+00:00",
+    "exported_at": "2025-02-05T17:40:05+01:00",
     "name": "Icarus-Dedicated",
     "author": "bolverblitz@ebg.pw",
     "uuid": "7f2f0676-3d0c-457f-a3aa-b2af40966698",
```

Comment:
Please check that, i do not have a pelican panel so i just copyed this in the exported file.

Reply:
You messed up. And wrote the pterodactyl exported egg over the pelican one. Please fix.

Reply:
Do not touch the once you do not have. We will take care about that.

Reply:
> Do not touch the once you do not have. We will take care about that.

So should i revert the file back to its original state or will you fix it?

Reply:
Yea just revert it.